src/transformers/modeling_flax_utils.py

```diff
@@ -425,6 +432,133 @@ def save_pretrained(self, save_directory: Union[str, os.PathLike], push_to_hub=F
             logger.info(f"Model pushed to the hub in this commit: {url}")
 
 
+class SequenceSummary(nn.Module):
```

Comment:
Didn't need to add this, but feel like it might be usefull when other classes get ported to Flax

Reply:
Could we maybe leave it out for now and only add it once needed? :-) I think it's not a good idea to have unused/dead code in the repo

Reply:
Maybe I was unclear, this is in fact used for now (see [this](https://github.com/huggingface/transformers/blob/f41e4db62662913e36a069910a97770da62b159f/src/transformers/models/electra/modeling_flax_electra.py#L564)). What I meant is that maybe I could have done it without introducing another class, but this way its "scales" more easily to new models

Reply:
The PyTorch `SequenceSummary` was a mistake and we will remove it at some point. It goes against our philosophy of having one independent file per model and no abstraction for modeling classes.

So I would not add this here and just define the needed class for Electra in the electra modeling file, ignore the `config.summary_type` argument and just implementing the "first" type. It should also be named with the `FlaxElectra` prefix.

Reply:
Makes alot of sense, I will move the class to the proper file and rename it (however I'll keep the variable name fore consistency between checkpoints)